azure-nextgen-py-appservice/Pulumi.yaml

```
@@ -0,0 +1,14 @@
+name: azure-nextgen-py-appservice
+runtime: python
+description: Creates Azure App Service with SQL Database and Application Insights in Python
+template:
+  config:
+    azure:environment:
+      description: The Azure environment to use (`public`, `usgovernment`, `german`, `china`)
+      default: public
+    azure:location:
```

Please check everywhere... Although, why do we have the template section in an example? Does that work?
```suggestion
    azure-nextgen:location:
```